docs/application/native/guides/machine-learning/machine-learning-inference.md

```diff
@@ -378,6 +431,70 @@ For more information about the pipeline states, see [GStreamer guide](https://gs
     ml_tensors_data_set_tensor_data (data, 0, dummy, 1);
     ```
 
+## Custom Filter
+
+For developers' convenience, NNStreamer provides an interface for processing the tensors data with `custom-easy` framework. After registering the user-defined callback function with input/output tensors information, it can manipulate tensors data in the pipeline without an independent shared object. Since callback function works as _filter_ in the pipeline, it is named as `Custom Filter`.
+
+- **Define and register Custom Filter**
+
+    Before using the Custom Filter in the pipeline, you have to register it with input/output tensors information and its name.
```

Comment:
We need to replace / with appropriate word: 

Doubt: 
What is it?   
input **or** output 
input **and** output 

What does "**its**" refer to? 
 ....its name ..

The following is just a suggestion, but kindly clear my doubts to help me understand better: 

```suggestion    
    Before using the Custom Filter in the pipeline, you have to register the Custom Filter with input and output tensor information and its name.
```
If the suggestion is fine, please verify for technical correctness

Reply:
`Custom Filter` is a kind of user-defined function which processes the tensors data in the pipeline. To use this, input __and__ output tensors information is required. Moreover, `Custom Filter` is internally managed by its name so a unique name is needed when registering the Custom Filter.

In my opinion, your suggestion is good for me.